packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Details/ForkliftControllerDetailsTab.tsx

```diff
@@ -18,13 +20,31 @@ interface ForkliftControllerDetailsTabProps extends RouteComponentProps {
 export const ForkliftControllerDetailsTab: React.FC<ForkliftControllerDetailsTabProps> = ({
   obj,
 }) => {
+  const [userSettings, hideWelcomeCard, setHideWelcomeCard] = useWelcomeStateByUserSettings();
+
+  // Set and use context data for the overview page state
+  const { setData } = useCreateOverviewStateContext();
+  const { data: { hideWelcomeCardByContext } = {} } = useCreateOverviewStateContext();
+
+  const isDisplayingWelcomeCard = () =>
+    hideWelcomeCardByContext !== undefined ? !hideWelcomeCardByContext : !hideWelcomeCard;
```

Comment:
combining the two hooks into one hook will make this one much nicer, there will be only one value from one hook

Reply:
Done